process/block/shardblock.go

```diff
@@ -272,7 +280,12 @@ func (sp *shardProcessor) ProcessBlock(
 		return err
 	}
 
+	startTime = time.Now()
 	err = sp.checkValidatorStatisticsRootHash(header, processedMetaHdrs)
+	elapsedTime = time.Now().Sub(startTime).Seconds()
+	log.Debug("elapsed time to check validator statistics root hash",
```

Comment:
i think these should be deleted.

Reply:
These are set on debug level anyway and could be removed after it would be confirmed that everything is ok as time consumption on these calls. Because of these helpful prints I caught an old error :)